### PR TITLE
README for the new flag.

### PR DESCRIPTION
**Pull request by jiqing-feng**
@echarlaix Hi, Ella. The README is better to be changed since I merged my codes. I marked that the flag **generate_teacher_logits** can save teacher logits to accelerate distillation training process. Would help me to review it? Thanks.

### CODE REVIEW

File: examples/neural_compressor/question-answering/README.md [hunk 1]
```diff
@@ -83,3 +83,18 @@ and [pruning](https://github.com/huggingface/optimum/blob/main/examples/config/p
 configuration files will be used.
 
 The flag `--verify_loading` can be passed along to verify that the resulting quantized model can be loaded correctly.
+
+The distillation process can be accelerated by reserving teacher logits.
```
Comment (echarlaix) [inline examples/neural_compressor/question-answering/README.md:87]: I would be in favor of adding `generate_teacher_logits` to the already existing command line presenting knowledge distillation for user to use it by default and to not have too many examples as well (to have a good readability)

**Review by echarlaix — COMMENTED** [review]
LGTM !

**Comment by jiqing-feng** [comment]
@echarlaix Hi Ella, I have removed the unnecessary command and add the flag to the existing command. Please check it, thanks

**Review by PenghuiCheng — APPROVED** [review]
